### ISSUE TITLE
Isn't the ground albedo a reflectance?

### ISSUE THREAD

**Issue by ekmett** [TheRealMJP/DeferredTexturing#2]
In the sample framework code `Skybox.cpp` you compute the spectrum for the ground albedo using

```
 SampledSpectrum groundAlbedoSpectrum = SampledSpectrum::FromRGB(Albedo);
```

By default PBRT assumes `FromRGB` is getting an illuminant, but isn't the ground albedo providing reflectance? I'd have expected

```
 SampledSpectrum groundAlbedoSpectrum = SampledSpectrum::FromRGB(Albedo,  SpectrumType::Reflectance);
```

**Comment by TheRealMJP** [comment]
Yes, I believe that this is a bug and your version is correct. It looks like the debug layer is catching some new issues with the latest Windows 10 update, so I'll fix this once I get the debug layer warnings straightened out. Thanks for pointing this out!

**Comment by TheRealMJP** [comment]
Fixed in commit [86a4677](https://github.com/TheRealMJP/DeferredTexturing/commit/86a4677de9082495d31d20bacfe5c7d5876afede)

**Comment by ekmett** [comment]
No, thank you! =)

I just wanted to say that all the work you put into providing demonstration code to go with your talks is very well appreciated.
